src/Std/Time/Zoned/ZoneRules.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+prelude
+import Std.Time.Time
+import Std.Time.DateTime
+import Std.Time.Internal
+import Std.Time.Zoned.Offset
+import Std.Time.Zoned.TimeZone
+
+namespace Std
+namespace Time
+namespace TimeZone
+open Internal
+
+/--
+Represents the type of local time in relation to UTC.
+-/
+inductive UTLocal
+  | ut
+  | local
+  deriving Repr, Inhabited
+
+/--
+Represents types of wall clocks or standard times.
+-/
+inductive StdWall
+  | wall
+  | standard
+  deriving Repr, Inhabited
+
+/--
+Represents a type of local time, including offset and daylight saving information.
+-/
+structure LocalTimeType where
```

Comment:
Is the TZ identifier (like `Asia/Tokyo`) available here? I think that it would be good to be able to query time zones by identifier.

Reply:
I'm going to add a field for the tz identifier.